gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.9.14",
+  "version": "2.9.16",
```

Comment:
Looks like a double-bump.  Double check that it's right after merging from master.

Reply:
> Looks like a double-bump. Double check that it's right after merging from master.

I see master as 2.9.15